python/tvm/relay/op/_transform.py

```diff
@@ -67,6 +67,10 @@
 _reg.register_injective_schedule("adv_index")
 
 
+@_reg.register_compute("concatenate")
+def compute_concat(attrs, inputs, output_type):
+    return [topi.concatenate(inputs, attrs.axis)]
```

Comment:
Right now this code doesn't work on GPU, because `topi.concatenate` is hard coded for x86. 

We should remove `register_compute("concatenate")` and instead introduce proper op strategy for `concat`. See how compute and schedule for other ops are registered in `relay/op/strategy`.

Reply:
Done